tests/util.py

```diff
@@ -14,8 +14,8 @@ def clean_xml(xml_str):
     Remove whitespace from XML.
     """
     parser = etree.XMLParser(remove_blank_text=True)
-    return etree.tostring(
-        etree.XML(xml_str, parser=parser))
+    return str(etree.tostring(
```

Comment:
I just read your other comment, maybe try `s.decode('utf-8')` instead? You may need to check the type or for an `AttributeError`